src/action.ts

```diff
@@ -280,27 +279,26 @@ class Action {
       }
     }
 
-    let executionGraphReport: ExecutionGraphReport | undefined = undefined
+    let executionGraphBundle 
 
     if (executionGraph.status === TaskStatus.Succeeded) {
       try {
-        executionGraphReport = await this.vib.getExecutionGraphReport(executionGraphId)
-        core.setOutput("result", executionGraphReport)
+        executionGraphBundle = await this.vib.getExecutionGraphBundle(executionGraphId)
 
-        if (!executionGraphReport.passed) {
-          core.setFailed(`Execution graph succeeded, however some tasks didn't pass the verification.`)
+        if (!executionGraphBundle.passed) {
```

Comment:
Note that the bundle is not a JSON like the report. Instead, we're downloading a `tar.gz`, so what we have to do is:
1. Download the bundle and put it somewhere. The way to download this file is like we're doing with the `raw-report`: `vib.ts` should set the `responseType: "stream"` config in the request, and the function should return a `Readable` instead of `ExecutionGraphBundle`, then that stream should be piped into a file, like its's done inside `#downloadRawReport`
2. Having downloaded the `.tar.gz`, we'll want to extract it, so we can use the extracted directory as `artifact`
3. If `executionGraph.status === TaskStatus.Succeeded`, we can access the execution graph report inside the extracted directory